launcher/preload.sh

```diff
@@ -7,7 +7,16 @@ copy_launcher() {
 setup_launcher_systemd_unit() {
   cp launcher/container-runner.service /usr/share/oem/container-runner.service
   # set attest service endpoint
-  sed -i 's/\${ATTEST_ENDPOINT}/'${ATTEST_ENDPOINT}'/g' /usr/share/oem/container-runner.service
+
+  if [ "$IMAGE_ENV" == "prod" ]; then
```

Comment:
let's go with "hardened", as that is more descriptive than prod. The prod image is relative to the user but from our POV both images could be prod images.